tests/projects/test_entry_point.py

```diff
@@ -15,17 +15,17 @@ def test_entry_point_compute_params():
     Tests that EntryPoint correctly computes a final set of parameters to use when running a project
     """
     project = load_project()
-    entry_point = project.get_entry_point("greeter")
+    entry_point = project.get_entry_point("greeter_compute_params")
```

Comment:
We should be adding tests that include cases where we don't want to include the action. We should also add in tests that check for failures when the action is used improperly to make sure the feature errors out as expected.

Reply:
Added more cases. Let me know if these are sufficient